docs/serving/samples/hello-world/helloworld-go/Dockerfile

```diff
@@ -1,24 +1,25 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.12 as builder
+FROM golang:1.13 as builder
 
 # Copy local code to the container image.
-WORKDIR /go/src/github.com/knative/docs/helloworld
-COPY . .
+WORKDIR /app
+COPY . ./
 
 # Build the command inside the container.
 # (You may fetch or manage dependencies here,
 # either manually or with a tool like "godep".)
 RUN CGO_ENABLED=0 GOOS=linux go build -v -o helloworld
```

Comment:
Imo “-o hello-server” is more descriptive. Or “-o app”

In such tutorials we end up with naming everything hello, so it’s often unclear what they refer to.

Reply:
Agreed.